automation/dependDockerfile/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM quay.io/openshift/origin-jenkins-agent-base:v4.0.0
+# Jenkins slave requirements
+RUN yum makecache \
+ && yum erase -y java-1.8.0-openjdk-headless \
+ && yum -y install openssl java-1.8.0-openjdk \
+ && pushd /etc/pki/ca-trust/source/anchors \
+ && curl -s -k -O https://password.corp.redhat.com/RH-IT-Root-CA.crt \
+ && curl -s -k -O https://password.corp.redhat.com/legacy.crt \
+ && curl -s -k -O https://engineering.redhat.com/Eng-CA.crt \
+ && update-ca-trust extract \
+ && openssl x509 -in RH-IT-Root-CA.crt -out RH-IT-Root-CA.der -outform der \
+ && openssl x509 -in legacy.crt -out legacy.der -outform der \
+ && openssl x509 -in Eng-CA.crt -out Eng-CA.der -outform der \
+ && export JAVA_HOME="/usr/lib/jvm" \
+ && export PATH="${PATH}:${JAVA_HOME}/jre/bin" \
+ && keytool -importcert -noprompt -alias RH-IT-Root-CA -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -file RH-IT-Root-CA.der \
+ && keytool -importcert -noprompt -alias legacy -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -file legacy.der \
+ && keytool -importcert -noprompt -alias Eng-CA -keystore $JAVA_HOME/jre/lib/security/cacerts -storepass changeit -file Eng-CA.der
+ENTRYPOINT ["/usr/local/bin/run-jnlp-client"]
+# General requirements                                                          
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+ && yum install -y gcc python-devel \
+ && curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" \
```

Comment:
once you have epel, you can install pip with `yum install python-pip`

Reply:
done